erpnext/accounts/report/consolidated_financial_statement/consolidated_financial_statement.py

```diff
@@ -115,7 +115,9 @@ def prepare_companywise_opening_balance(asset_data, liability_data, equity_data,
 		# opening_value = Aseet - liability - equity
 		for data in [asset_data, liability_data, equity_data]:
 			account_name = get_root_account_name(data[0].root_type, company)
-			opening_value += get_opening_balance(account_name, data, company)
+			value = get_opening_balance(account_name, data, company)
+			if value:
+				opening_value += value
```

Comment:
Not _required_, but I guess it is more succinct.

```suggestion
			opening_value += get_opening_balance(account_name, data, company) or 0
```